tests/framework/installer/cassandra_manifests.go

```diff
@@ -0,0 +1,279 @@
+package installer
+
+import (
+	"fmt"
+	cassandrav1alpha1 "github.com/rook/rook/pkg/apis/cassandra.rook.io/v1alpha1"
+)
+
+type CassandraManifests struct{}
+
+func (i *CassandraManifests) GetCassandraCRDs() string {
+	return `apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: clusters.cassandra.rook.io
+spec:
+  group: cassandra.rook.io
+  names:
+    kind: Cluster
+    listKind: ClusterList
+    plural: clusters
+    singular: cluster
+  scope: Namespaced
+  version: v1alpha1`
+}
+
+func (i *CassandraManifests) GetCassandraOperator(namespace string) string {
+
+	return fmt.Sprintf(`
+# Namespace where Cassandra Operator will live
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: %[1]s
+
+---
+# ClusterRole for cassandra-operator.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: rook-cassandra-operator
+rules:
+  - apiGroups:
+    - ""
+    resources:
+      - pods
+    verbs:
+      - get
+      - list
+      - watch
+      - delete
+  - apiGroups:
+      - ""
+    resources:
+      - services
+      - serviceaccounts
+    verbs:
+      - get
+      - list
+      - watch
+      - create
+  - apiGroups:
+      - ""
+    resources:
+      - persistentvolumes
+      - persistentvolumeclaims
+    verbs:
+      - get
+      - delete
+  - apiGroups:
+      - ""
+    resources:
+      - nodes
+    verbs:
+      - get
+  - apiGroups:
+      - rbac.authorization.k8s.io
```

Comment:
and also remove this granting of perms to the `rbac.authorization.k8s.io` API group too please.

Reply:
@jbw976 I had forgotten about the `cassandra_manifests.go` file, sorry about that! 
I believe I added the license header to all the files now. 
If there's nothing else and this build gets a green light (which it should since no code was changed) then we can merge !